models/recharge__charge_line_item_history.sql

```diff
@@ -94,7 +94,7 @@ with charges as (
 ), joined as (
     select
         unioned.charge_id,
-        unioned.index,
+        unioned.index, 
```

Comment:
This seems to be "0" for all of the non charge line items. This reads a bit confusing as the grain of the table seems to be off we duplicate indexes of 0.

Can we create a new column that is a row_num of the charge_id to show the grain a bit clearer. Or should we rethink the creation of these index values?